test/extended/authentication.sh

```diff
@@ -14,8 +14,6 @@ source "${OS_ROOT}/hack/common.sh"
 os::log::install_errexit
 cd "${OS_ROOT}"
 
-ensure_iptables_or_die
```

Comment:
I seem to have misunderstood something rather fundamental.  I thought `iptables` was required to make services work, so this script would be unable to use the IP it finds here: https://github.com/openshift/origin/pull/5447/files#diff-cbfdb4abd12b4cc83e7ccde4c32d0281R96